dev/InfoBar/TestUI/InfoBar.cs

```diff
@@ -199,33 +205,21 @@ public object ActionButtonCommandParameter
         /* Close Button Properties
          * 
          */
-        public object CloseButtonContent
-        {
-            get { return (object)GetValue(CloseButtonContentProperty); }
-            set { SetValue(CloseButtonContentProperty, value); }
-        }
-
-        public static readonly DependencyProperty CloseButtonContentProperty =
-            DependencyProperty.Register(nameof(CloseButtonContent), typeof(object), typeof(InfoBar), new PropertyMetadata(null, OnPropertyChanged));
-
-        public Style CloseButtonStyle
```

Comment:
>CloseButtonStyle [](start = 21, length = 16)

Rather than setting these margins from code behind we should be calling GoToState so that these values are customizable

Reply:
Good idea - I created some VisualStates that modify the margins depending on what visual components are visible instead of explicitly setting the margins in my InfoBar.cs.